fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -7295,6 +7295,13 @@ public ParseNode visitSimpleFunctionCall(StarRocksParser.SimpleFunctionCallConte
             }
         }
 
+        // support regexp_replace with 2 param
+        if (functionName.equalsIgnoreCase(FunctionSet.REGEXP_REPLACE) && context.expression().size() == 2) {
```

Comment:
Do not support regexp_replace for SR in this PR, you can submit another PR for this